spec/acceptance/class_spec.rb

```diff
@@ -2,7 +2,11 @@
 
 require 'spec_helper_acceptance'
 
-describe 'puppetboard class' do
+RSpec.configure do |c|
+  c.filter_run_excluding "broken_#{fact('os.name')}#{fact('os.release.major')}_puppet#{ENV['BEAKER_PUPPET_COLLECTION'].sub(%r{puppet}, '')}".downcase.to_sym => true
+end
+
+describe 'puppetboard class', broken_debian11_puppet6: true do
```

Comment:
In most places we use `if:` or `unless:`, so:
```suggestion
describe 'puppetboard class', `unless: fact('os.name') == 'Debian' && `fact('os.release.major') == '11' do
```

For brevity I didn't specify the Puppet version.

Another thing to consider is what I did for theforeman-puppet: there's `unsupported_puppetserver` as a helper method: https://github.com/theforeman/puppet-puppet/blob/master/spec/support/acceptance/puppetserver.rb

Then we use `unless: unsupported_puppetserver` (https://github.com/theforeman/puppet-puppet/blob/84492f25379319c09d2fdcb26aa4e331fba4170c/spec/acceptance/puppetserver_latest_spec.rb#L3=).

Thinking more about it, perhaps it shouldn't be negatively named. So rather `if: has_puppetserver` instead. However, that's an implementation detail.

Reply:
The Foreman approach looks gorgeous!  :heart: